src/pages/Account/AccountPlaceholder/index.tsx

```diff
@@ -0,0 +1,33 @@
+/** @jsxImportSource @emotion/react */
+
+import { Typography } from "@mui/material";
+import { PrimaryButton } from "components";
+
+import { useTranslation } from "translation";
+import wallet from './wallet.png';
+import { useStyles } from './styles';
+
+const AccountPlaceholder: React.FC = () => {
+  const { t } = useTranslation();
+  const styles = useStyles();
+
+  return (
+    <div css={styles.placeholderContainer}>
+      <div>
+        <img
+          src={wallet}
+          css={styles.wallet}
+          alt={t('dashboard.connectWalletBanner.illustration.alt')}
+        />
+      </div>
+      <Typography css={styles.placeholderText} variant='h4'>
+        {t('accountPlaceholder.assetsWillAppearHere')}
+      </Typography>
+      <PrimaryButton>
+        {t('accountPlaceholder.letsGetStarted')}
+      </PrimaryButton>
```

Comment:
@therealemjy where should this button take us to?

Reply:
Oh yeah, that should have been mentioned: the dashboard page.